Modules/TPC/run/tpcQCTracks_sampled.json

```diff
@@ -31,20 +31,24 @@
         "dataSource_comment": "The other type of dataSource is \"direct\", see basic-no-sampling.json.",
         "dataSource": {
           "type": "dataSamplingPolicy",
-          "name": "tpc-tracks"
+          "name": "tpc-tracks",
```

Comment:
hello Max! This won't work, you need:
```
      "dataSources": [{
        "type": "dataSamplingPolicy",
        "name": "tpc-tracks"
      }, {
        "type": "direct",
        "query": "meanvertex:CTP/MEANVERTEX/0?lifetime=condition&ccdb-path=CTP/Calib/MeanVertex"
      }],
```

Reply:
I see thanks a lot Piotr! I will adapt and push an update.